client/src/components/Reviews.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EachReview from './EachReview.jsx';
+
+const Reviews = ({ reviews }) => (
+  <div>
+    {reviews.map((review) => <EachReview key={review.id} eachReview={review} />)}
+  </div>
```

Comment:
Do lines 6-8 render properly? It looks fine to me but I am not sure why the two divs are highlighted differently. Usually they match in style.

Reply:
Hmm that's strange! It isn't mismatched in my vscode and does render as I expect it to, but that is definitely not matching here. Weird.